lib/Conversion/TTNNToEmitC/Utils.cpp

```diff
@@ -4,10 +4,55 @@
 
 #include "ttmlir/Conversion/TTNNToEmitC/Utils.h"
 
+#include "mlir/Dialect/EmitC/IR/EmitC.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
 #include "mlir/Pass/Pass.h"
 
 namespace mlir::tt::ttnn_to_emitc::utils {
 
+mlir::ModuleOp getParentModule(mlir::Operation *op) {
```

Comment:
nit: This one is for debate...

Should we copy\paste function comments from .h files?

It would be nice if I didn't need to look at the header to get some context. :)

Reply:
I actually went and googled what people think on this topic, and when I compiled it in my head, I think I agree with this the most: https://stackoverflow.com/a/3572839

High-level description of `what the function does` goes to header file, while the description of `how the function does what it does and why it does it that way` goes to the implementation file.

That said, I'll add some implementation specific comments.

Reply:
Great! Thanks Sale!